specification/cognitiveservices/data-plane/CustomVision/Prediction/readme.md

```diff
@@ -20,6 +20,13 @@ These settings apply only when `--tag=release_1_0` is specified on the command l
 input-file: stable/v2.0/Prediction.json
 ```
 
+### Release 3.0
+These settings apply only when `--tag=release_3_0` is specified on the command line.
+
+``` yaml $(tag) == 'release_3_0'
+input-file: stable/v3.0/Prediction.json
+```
+
```

Comment:
Thanks, added. First attempted failed because it looks like the build doesn't actually use the package name. Or expects a 3 named package and is mangling our 4 parts. For example, Our packages are azure_cognitiveservices_customvision_training but the build is mangling the end to azure_cognitiveservices_customvisiontraining  so I updated our readme.ruby.md files accordingly.